build/Dockerfile.functest

```diff
@@ -4,13 +4,20 @@ WORKDIR /go/src/github.com/kubevirt/hyperconverged-cluster-operator/
 COPY . .
 RUN make build-functest
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi8/ubi
 
 ENV USER_UID=1001 \
     TEST_OUT_PATH=/test
 
 WORKDIR ${TEST_OUT_PATH}
 ENTRYPOINT ./hack/run-tests.sh
+
+RUN curl -LO https://dl.k8s.io/release/v1.20.0/bin/linux/amd64/kubectl && \
```

Comment:
should we use openshift-client instead of kubectl?
```
curl https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz -o openshift-client-linux.tar.gz && \
tar -xvzf openshift-client-linux.tar.gz -C /usr/local/bin/ && \
rm -rf openshift-client-linux.tar.gz
```

Reply:
otherwise if we prefer kubectl, the reccomended path shoudl be:
```
RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" && \
chmod a+x ./kubectl && \
mv ./kubectl /usr/local/bin
```

Reply:
I prefered kubectl since
- openshift-client is tar.gz file whereas tar command does not exist in ubi-minimal
- current tests work with kubectl.